apps/client/src/views/cuesheet/cuesheet-table-elements/EditableCell.tsx

```diff
@@ -36,18 +45,35 @@ const EditableCell = (props: EditableCellProps) => {
     setValue(initialValue);
   }, [initialValue]);
 
+  if (isMultiLine) {
```

Comment:
question: do you think it is worth making two components? it seems like we want quite a bit of logic to be different?